.github/workflows/github_release.yaml

```diff
@@ -0,0 +1,80 @@
+name: GitHub Release Attestation
+
+on:
+  release:
+
+permissions:
+  contents: read
+
+jobs:
+  # This reusable workflow inspects if the given workflow_name exists on Chainloop. If the Workflow does not exist
+  # it will create one with an empty contract ready for operators to be filled. Otherwise, if found, it will just
+  # be ignored and the process will continue. For this to work it's using a pre-created API Token
+  onboard_workflow:
+    name: Onboard Chainloop Workflow
+    uses: chainloop-dev/labs/.github/workflows/chainloop_onboard.yml@4173e015dbd5dc2a8802555c268da63d57bbe576
+    with:
+      project: "chainloop"
+      workflow_name: "chainloop-github-release"
```

Comment:
what would happen if you remove the name?

Reply:
It will be created automagically.